src/matrix/mod.rs

```diff
@@ -334,6 +350,8 @@ mod tests {
 
     #[tokio::test]
     async fn send_text_message() {
+        //crate::CONTEXT.scope(std::cell::RefCell::new(None),  async {}).await;
```

Comment:
What is this doing here?

Reply:
This is a method to set the state for `crate::CONTEXT` when using `tokio::task_local!` as a replacement for `futures::task_local!` since the `tokio` version doesn't provide default state. I never actually got this to work, so i switched to `std::thread_local!`. There's an unresolved discussion about this above, but for now ill remove this dead code.